src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/storm/bolt/uniisl/UniIslHandler.java

```diff
@@ -122,11 +130,22 @@ public void notifyBfdStatus(Endpoint endpoint, IslReference reference, BfdStatus
         emit(getCurrentTuple(), makeDefaultTuple(new IslBfdStatusUpdateCommand(endpoint, reference, status)));
     }
 
+    @Override
+    public void slowPollUpdateRequest(Endpoint endpoint, boolean enableSlowPoll) {
+        WatchListCommand command = new WatchListSlowPollUpdateCommand(endpoint, enableSlowPoll);
+        emit(STREAM_POLL_ID, getCurrentTuple(), makePollTuple(command));
```

Comment:
Same here, regarding loops.

Reply:
This anchor has been removed in this place.